python/ray/ray_constants.py

```diff
@@ -192,6 +192,11 @@ def to_memory_units(memory_bytes, round_up):
 # A constant used as object metadata to indicate the object is raw bytes.
 OBJECT_METADATA_TYPE_RAW = b"RAW"
 
+# A constant used as object metadata to indicate the object is an actor handle.
+# TODO(fyrestone): Serialize the ActorHandle viathe custom type feature
```

Comment:
```suggestion
# TODO(fyrestone): Serialize the ActorHandle via the custom type feature
```
Can you also document that this needs to be in sync with the Java definition in the xxx.java file?

Reply:
The comments has been updated.